examples/react/react-router/src/index.jsx

```diff
@@ -27,38 +27,46 @@ const queryClient = new QueryClient({
   },
 });
 
+const router = createBrowserRouter([
+    {
+      path: "/",
+      element: <Root />,
+      errorElement: <ErrorPage />,
+      loader: rootLoader(queryClient),
+      action: rootAction(queryClient),
+      children: [
+        {
+          index: true,
+          element: <Index />
+        },
+        {
+          path: "contacts/:contactId",
+          element: <Contact />,
+          loader: contactLoader(queryClient),
+          action: contactAction(queryClient),
+        },
+        {
+          path: "contacts/:contactId/edit",
+          element: <EditContact />,
+          loader: contactLoader(queryClient),
+          action: editAction(queryClient),
+        },
+        {
+          path: "contacts/:contactId/destroy",
+          element: <EditContact />,
+          action: destroyAction(queryClient),
+          errorElement: <div>Oops! There was an error.</div>
+        },
+      ],
+    },
+  ],
+);
```

Comment:
alternative approach:
```suggestion
const router = createBrowserRouter(
  createRoutesFromElements(
    <Route
      path="/"
      element={<Root />}
      errorElement={<ErrorPage />}
      loader={rootLoader(queryClient)}
      action={rootAction(queryClient)}
    >
      <Route index element={<Index />} />
      <Route
        path="contacts/:contactId"
        element={<Contact />}
        loader={contactLoader(queryClient)}
        action={contactAction(queryClient)}
      />
      <Route
        path="contacts/:contactId/edit"
        element={<EditContact />}
        loader={contactLoader(queryClient)}
        action={editAction(queryClient)}
      />
      <Route
        path="contacts/:contactId/destroy"
        action={destroyAction(queryClient)}
        errorElement={<div>Oops! There was an error.</div>}
      />
    </Route>
  )
);
```